test_files/coerce.tr.js

```diff
@@ -1 +1 @@
-let x = 'hello, ' + (JSON.parse('"world"'));
+let x = 'hello, ' + JSON.parse('"world"');
```

Comment:
Note that this output is still wrong (we lost the Closure type), but that is due to the TS bug where comments are dropped.